bin/i18n/resources/apps/animations/sync_out.rb

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env ruby
+
+require 'fileutils'
+require 'json'
+
+require_relative '../../../../animation_assets/manifest_builder'
+require_relative '../../../i18n_script_utils'
+require_relative '../animations'
+
+module I18n
+  module Resources
+    module Apps
+      module Animations
+        class SyncOut
+          def self.perform
+            new.execute
+          end
+
+          def execute
+            pegasus_languages.each.with_index(1) do |pegasus_lang, next_lang_idx|
+              progress_bar.title = progress_bar_title(next_lang_idx)
+
+              crowdin_spritelab_file_path = CDO.dir(File.join(I18N_LOCALES_DIR, pegasus_lang[:crowdin_name_s], DIR_NAME, SPRITELAB_FILE_NAME))
+              next unless File.exist?(crowdin_spritelab_file_path)
```

Comment:
Only files with updates are downloaded, therefore we want to skip any action on languages with no updates.